.gitignore

```diff
@@ -1,9 +1,7 @@
 # Compiled files
 dist/
-source/_data/env.json
-source/_patterns/01-atoms/icon/font/
-source/_patterns/01-atoms/icon/scss/_icon-map-generated.scss
-source/_patterns/01-atoms/svgicon/scss/_icons-generated.scss
+env.json
+_icons-generated.scss
```

Comment:
do these two still need the source/default/ path? I see an env.json in source/default/_data/env.json but not at root